app/assets/v2/js/cart.js

```diff
@@ -1023,7 +1022,7 @@ Vue.component('grants-cart', {
         };
       }
 
-      return this.filterByChainId.filter(token => token.name === name && token.networkId == this.networkId)[0];
+      return this.filterByChainId.filter(token => token.name === name && token.network == this.network)[0];
```

Comment:
polygon mainnet has

network: mainnet
networkId: 137

filtering by network may return the wrong token

Reply:
Ahh good call - annoyingly we've set all rinkeby tokens up with a `networkId` of 1 - I think we can work around that though 🙌